src/Commands/ListViewCommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Sven\ArtisanView\Commands;
+
+use Illuminate\Console\Command;
+use Sven\ArtisanView\View;
+
+class ListViewCommand extends Command
+{
+    /**
+    * The name and signature of the console command.
+    *
+    * @var string
+    */
+   protected $signature = 'list:view';
+
+   /**
+    * The console command description.
+    *
+    * @var string
+    */
+   protected $description = 'Prints the directory structure of the views directory';
+
+   /**
+    * Create a new command instance.
+    */
+   public function __construct()
+   {
+       parent::__construct();
+   }
+
+   /**
+    * Execute the console command.
+    *
+    * @return mixed
+    */
+   public function handle()
```

Comment:
Are you saying that I should make a new class within the  Sven\ArtisanView\Tests namespace containing the tests for the list command? Also, isn't it cleaner if the logic of the command recursively checks directories and returns the proper output, and the handling determines what to do with the return value? I could try making getListAsString private and call handle in the tests if that's what you mean.